refs/types_test.go

```diff
@@ -18,3 +19,32 @@ func TestZeroSubnet(t *testing.T) {
 	refs.MakeZeroSubnet(id)
 	require.True(t, refs.IsZeroSubnet(id))
 }
+
+func TestSubnetID_MarshalText(t *testing.T) {
+	var id refs.SubnetID
+
+	const val = 15
+
+	id.SetValue(val)
+
+	txt, err := id.MarshalText()
+	require.NoError(t, err)
+
+	res, err := strconv.ParseUint(string(txt), 10, 32)
+	require.NoError(t, err)
+
+	require.EqualValues(t, val, res)
+}
+
+func TestSubnetID_UnmarshalText(t *testing.T) {
```

Comment:
Some error tests are also desirable, at least overflow one (`math.MaxUint32+1`)

Reply:
Tested and documented.